src/launcher_config.py

```diff
@@ -96,6 +96,14 @@ def audio_volume(self) -> int:
     def audio_volume(self, value: int) -> None:
         self._data["audio"]["volume"] = min(max(value, 1), 20)
 
+    @property
+    def audio_volume_override_danger(self) -> int:
+        return int(self._data["audio"].get("volume_override_danger", 12))
+    
+    @audio_volume_override_danger.setter
+    def audio_volume(self, value: int) -> None:
```

Comment:
function name here is accidentally `audio_volume` but should be `audio_volume_override_danger`

Reply:
Nice catch, thanks. Funny that it worked with the wrong function name :shrug: